src/main/java/me/namuhuchutong/step3/RacingRule.java

```diff
@@ -0,0 +1,6 @@
+package me.namuhuchutong.step3;
+
+public interface RacingRule {
+
+    boolean isSatisfiedRule();
+}
```

Comment:
```suggestion

@FunctionalInterface
public interface RacingRule {

    boolean isSatisfiedRule();
}
```

인터페이스에 추상 메서드가 하나만 있네요
메서드가 하나만 존재 할 때 추론이 가능하기 때문에 람다식을 사용하여 객체를 넘길 수 있는데요
이럴 때 `@FunctionalInterface`을 붙여주시면 더 좋을 것 같아요